packages/compass-web/polyfills/@mongodb-js/saslprep/index.ts

```diff
@@ -0,0 +1,7 @@
+export function saslprep(): void {
+  // no-op
```

Comment:
nit: I'd make this throw instead of a no-op, just so the error message is nicer if the driver (or some other future package in Compass) at some point does try to call this